aspnetcore/security/authorization/iauthorizationpolicyprovider.md

```diff
@@ -141,7 +141,7 @@ Then, the `GetPolicyAsync` method can be updated to use the `BackupPolicyProvide
 
 ```csharp
 ...
-return BackupPolicyProvider.GetPolicyAsync(policyName);
```

Comment:
Since we're returning a task, there's no need to await it. We can return the task directly and our callers can await _that_ task rather than awaiting our own method (which, in turn, awaits `BackupPolicyProvider.GetPolicyAsync`). Plus, avoiding an extra await will result in the compiler generating more efficient code.